code/game/gamemodes/dynamic/dynamic.dm

```diff
@@ -300,11 +310,11 @@ GLOBAL_VAR_INIT(dynamic_forced_threat_level, -1)
 
 	admin_panel() // Refreshes the window
 
-// Checks if there are HIGH_IMPACT_RULESETs and calls the rule's round_result() proc
+// Checks if there are NO_OTHER_ROUNDSTARTSs and calls the rule's round_result() proc
 /datum/game_mode/dynamic/set_round_result()
 	// If it got to this part, just pick one high impact ruleset if it exists
 	for(var/datum/dynamic_ruleset/rule in executed_rules)
-		if(rule.flags & HIGH_IMPACT_RULESET)
+		if(CHECK_BITFIELD(rule.flags, NO_OTHER_ROUNDSTARTS_RULESET))
```

Comment:
problem with this change is xenos and blob have round results but are high impact midrounds

Reply:
Alright, reverted.